x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/oidc/OpenIdConnectAuthenticateResponse.java

```diff
@@ -19,14 +19,16 @@
     private String accessTokenString;
     private String refreshTokenString;
     private TimeValue expiresIn;
+    private String realmName;
     private Authentication authentication;
 
     public OpenIdConnectAuthenticateResponse(Authentication authentication, String accessTokenString, String refreshTokenString,
-                                             TimeValue expiresIn) {
+                                             TimeValue expiresIn, String realmName) {
```

Comment:
I think we can save this new parameter because the realm information is already available via `authentication.getSourceRealm()`.

Reply:
Revert the changes for OIDC authenticate, as you point out - no need for this anymore.